queries/php_only/highlights.scm

```diff
@@ -265,12 +286,17 @@
   "protected"
   "public"
   "readonly"
-  "static"
+  (static_modifier)
+  (function_static_declaration
+    "static")
 ] @keyword.modifier
 
 [
   "return"
+  "exit"
   "yield"
+  (yield_expression
+    "from")
 ] @keyword.return
```

Comment:
I think you meant to highlight `from` in this case?

Reply:
"from" can only be used in the context of "yield from"---and php technically considers "yield from" a single keyword

Reply:
This is fixed now!